compiler/rustc_error_codes/src/error_codes/E0476.md

```diff
@@ -0,0 +1,21 @@
+The value being coerced to does not outlive the coerced type.
```

Comment:
I think this is backwards. You always coerce from something that lives for 'long to something that lives for 'short

Reply:
Oops, this hurts my brain. I think that the `'b: 'a` is correct though.